src/som/interpreter/Method.java

```diff
@@ -0,0 +1,67 @@
+package som.interpreter;
+
+import com.oracle.truffle.api.CompilerDirectives;
+import com.oracle.truffle.api.TruffleLanguage;
+import com.oracle.truffle.api.frame.VirtualFrame;
+
+import som.compiler.ProgramDefinitionError;
+import som.vmobjects.SAbstractObject;
+import som.vmobjects.SMethod;
+
+public final class Method extends Invokable {
+
+
+    @CompilerDirectives.CompilationFinal(dimensions = 1) final byte[] bytecodes;
+    private final SMethod method;
+
+    public Method(final TruffleLanguage<?> language, final int numberOfBytecodes, final SMethod method) {
+        super(language);
+        this.bytecodes = new byte[numberOfBytecodes];
+        this.method = method;
+    }
+
+    @Override
+    public Object execute(final VirtualFrame frame) throws ReturnException {
+        Interpreter interpreter = (Interpreter) frame.getArguments()[0];
+        Frame newFrame = (Frame) frame.getArguments()[1];
+
+            try {
+                SAbstractObject result = interpreter.start(newFrame);
+                return result;
+            } catch (ReturnException e) {
+                if (e.hasReachedTarget(newFrame)) {
+                    SAbstractObject result = e.getResult();
+                    return result;
+                }
+                throw e;
+            } catch (ProgramDefinitionError programDefinitionError) {
+                programDefinitionError.printStackTrace();
+            }
+            return null;
+        }
```

Comment:
This method won't pass CI and the formatting requirements.
Perhaps you can somehow setup InteliJ to do a format on save automatically, using the Eclipse formatting settings.
I am pretty sure there's a way, since the Graal/Truffle team seems to use it.